unit_tests/library/algorithm.cpp

```diff
@@ -15,14 +15,6 @@ BOOST_AUTO_TEST_CASE(test_incompatible_with_mld)
         osrm::exception);
 }
 
-BOOST_AUTO_TEST_CASE(test_incompatible_with_corech)
```

Comment:
Same here we still need this test to make sure the fallback works.

Reply:
Added this test back, but renamed it.